src/GR.jl

```diff
@@ -0,0 +1,5 @@
+plot(x::AbstractBlock; figsize=(10cm, 1cm), fontsize=10) = GR(context(0, 0, figsize...),
```

Comment:
can we have a working Compose based implementation first? Try not to commit everything in one single PR, it's not a good practice.